### ISSUE TITLE
Create method in utils to check if beta

### ISSUE THREAD

**Issue by domdomegg** [commons-app/apps-android-commons#2215]
**Summary:** 

Checking whether the current BuildConfig flavour is beta is done frequently. While it's extremely simple (`BuildConfig.FLAVOR.equals("beta")`) think it'd be nice to have all checks done to a method. There are also many variants (e.g. `==`, `equalsIngoreCase`) + duplicated helper methods already so would be good to centralise.

This would allow us to do 'find usages' to quickly see where the prod and beta build differ in the Java code.

**Comment by domdomegg** [comment]
Another benefit is it will stop Android Studio annoying me by suggesting parts are redundant
```
Condition 'BuildConfig.FLAVOR.equals("beta")' is always 'true'
```

**Comment by domdomegg** [comment]
Ideally to be done after #2216 merged to avoid conflicts

**Comment by sup95** [comment]
I would like to work on this!

**Comment by korde96** [comment]
Is this issue still open? I have made the changes on my local commit, so is it alright if I make a PR?

**Comment by domdomegg** [comment]
It is still open but I think @sup95 was working on it first. You can submit a pr but in future please try to avoid working on issues other people already are :)
